bors/src/git.rs

```diff
@@ -212,6 +213,10 @@ impl Git {
     }
 
     pub fn with_ssh(mut self, ssh_key_file: &Path) -> Self {
+        // if ssh key file is not found, do not continue
+        if !metadata(ssh_key_file).unwrap().is_file() {
+            panic!("SSH Key File is not a file")
+        }
```

Comment:
I can't comment on the line I would like to (github won't let me) but this should should be done on the fully constructed path, as that's what's actually going to be interpreted by the shell. We can also change this to be a bit cleaner and remove the unnecessary unwrap to the following:

```
if !path.is_file() {
    panic!(...);
}
```